src/slices/AccountSlice.ts

```diff
@@ -268,7 +277,7 @@ export const calculateUserBondDetails = createAsyncThunk(
 
 export interface IAccountSlice extends IUserAccountDetails, IUserBalances {
   giving: { sohmGive: number; donationInfo: IUserDonationInfo };
-  redeeming: { sohmRedeemable: number; recipientInfo: IUserRecipientInfo };
+  redeeming: { sohmRedeemable: string; recipientInfo: IUserRecipientInfo };
```

Comment:
Can you explain why this being a number was problematic? Other code recognises it as a number and hasn't created any problems.

https://github.com/OlympusDAO/olympus-frontend/blob/60bec6a7da8f888c9722221073301d83186b8535/src/views/Give/RedeemYield.tsx#L39

Reply:
@0xLienid can you check out the comments?

Reply:
not 100% sure it was causing problems as it seems that the real issues were because of zapslice, but the ethers.utils.formatUnits returns a string I believe so it seems best to make the typing match

Reply:
Are these changes still relevant?